apps/client/src/pages/builder/sidebars/left/dialogs/skills.test.tsx

```
@@ -0,0 +1,104 @@
+
+vi.mock('../sections/shared/section-dialog', () => ({
+  SectionDialog: ({ children }: any) => <div data-testid="section-dialog">{children}</div>,
+}));
+
+describe('SkillsDialog', () => {
+  it('renders without crashing', () => {
+    const { container } = render(<SkillsDialog />);
+    expect(container).toBeTruthy();
+  });
+
+  it('renders basic form elements', () => {
```

ESLint

[error] String not marked for translation. Wrap it with t``, <Trans>, or msg``.

(lingui/no-unlocalized-strings)